content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -197,23 +197,72 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
+
+There is also a known issue with containerd when trying to connect to SMB shares from Pods. Once you have configured GMSA, the pod will be unable to connect to the share using the Hostname or FQDN but connecting via IP address works as expected. 
+
+```PowerShell
+ping adserver.ad.local
+```
+Resolves the Hostname to IP Address Correctly and will produce an output similar to this:
+```
+```PowerShell
+Pinging adserver.ad.local [192.168.111.18] with 32 bytes of data:
+Reply from 192.168.111.18: bytes=32 time=6ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+```
+
+However, when attempting to browse the directory using the hostname
+
+```PowerShell
+cd \\adserver.ad.local\test
+```
+
+It will appear that the share doesn't exists:
+
+```PowerShell
+cd : Cannot find path '\\adserver.ad.local\test' because it does not exist.
+At line:1 char:1
++ cd \\adserver.ad.local\test
++ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : ObjectNotFound: (\\adserver.ad.local\test:String) [Set-Location], ItemNotFoundException
+    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand
+```
+
+While trying to browse to the share using the IP Address instead:
+
+```PowerShell
+cd \\192.168.111.18\test
+```
+
+Changes to the directory as expected and you should see a prompt similar to this:
+```PowerShell
+Microsoft.PowerShell.Core\FileSystem::\\192.168.111.18\test>
+```
+
+To correct the behaviour you must run the following `reg add "HKLM\SYSTEM\CurrentControlSet\Services\hns\State" /v EnableCompartmentNamespace /t REG_DWORD /d 1` and restart the pods to add the required registry key.
+
 ## Troubleshooting
 
 If you are having difficulties getting GMSA to work in your environment, there are a few troubleshooting steps you can take.
 
-First, make sure the credspec has been passed to the Pod.  To do this you will need to `exec` into one of your Pods and check the output of the `nltest.exe /parentdomain` command.  In the example below the Pod did not get the credspec correctly:
+First, make sure the credspec has been passed to the Pod.  To do this you will need to `exec` into one of your Pods and check the output of the `nltest.exe /parentdomain` command.  
 
-```shell
-kubectl exec -it iis-auth-7776966999-n5nzr powershell.exe
-
-Windows PowerShell
-Copyright (C) Microsoft Corporation. All rights reserved.
+In the example below the Pod did not get the credspec correctly:
 
-PS C:\> nltest.exe /parentdomain
+```PowerShell
+kubectl exec -it iis-auth-7776966999-n5nzr powershell.exe
+```
+`nltest.exe /parentdomain` results in the following error:
+```PowerShell
```

Comment:
@perithompson please omit the “PowerShell” from this line - it's not something you type into a shell.